packages/kbn-management/cards_navigation/src/consts.tsx

```diff
@@ -63,6 +63,13 @@ export const appDefinitions: Record<AppId, AppDefinition> = {
     }),
     icon: 'logstashQueue',
   },
+  [AppIds.DATA_QUALITY]: {
+    category: appCategories.DATA,
+    description: i18n.translate('management.landing.withCardNavigation.dataQualityDescription', {
+      defaultMessage: 'Manage the quality of your logs data.',
```

Comment:
@mdbirnstiehl do you have any recommendation for the description or does this looks good?

Reply:
I would say "log data" instead of "logs data". It's ok as is, but maybe we could be more descriptive of what users can actually do from the page with something like "Find and manage quality issues in your log data".

Reply:
cc @awahab07